test/slack-send-test.js

```diff
@@ -105,6 +105,30 @@ describe('slack-send', () => {
         assert.equal(chatArgs.oliver, 'benji', 'Correct message provided to postMessage');
         assert.equal(chatArgs.actor, 'user123', 'Correct message provided to postMessage');
       });
+      it('should accept a payload-file-path and use it\'s content in the message without replacing anything', async () => {
+        // Prepare
+        fakeCore.getInput.withArgs('channel-id').returns('C123456');
```

Comment:
Nit: I know this is in the other `payload-file-path` test, but I don't think we need to include `channel-id` here.